docs/source/ProgrammingGuide/Initialization.md

```diff
@@ -43,66 +45,68 @@ Kokkos chooses the two spaces using the following list:
-
-    --kokkos-threads=4 --threads=2
-
-set the number of threads to 4, while
+[`Kokkos::initialize`](../API/core/initialize_finalize/initialize) parses the command line for flags prefixed with `--kokkos-`, and removes all recognized flags.. Argument options are given with an equals (`=`) sign. If the same argument occurs more than once, the last one is used. For example, the arguments
 
-    --kokkos-threads=4 --threads=2 --kokkos-threads=3
+    --kokkos-threads=4 --kokkos-threads=3
 
 set the number of threads to 3. Table 5.1 gives a full list of command-line options.
 
 <h4>Table 5.1: Command-line options for Kokkos::initialize</h4>
 
 Argument | Description
 :---      | :---
---kokkos-help  <br/> --help   | print this message
---kokkos-threads=INT <br/> --threads=INT  | specify total number of threads or number of threads per NUMA region if used in conjunction with `--numa` option.
---kokkos-numa=INT <br/> --numa=INT | specify number of NUMA regions used by process.
---device-id=INT | specify device id to be used by Kokkos.
---num-devices=INT[,INT] | used when running MPI jobs. Specify the number of devices per node to be used. Process to device mapping happens by obtaining the local MPI rank and assigning devices round-robin. The optional second argument allows for an existing device to be ignored. This is most useful on workstations with multiple GPUs, one of which is used to drive screen output.
+--kokkos-help <br/> --help   | print this message
+--kokkos-disable-warnings    | disable kokkos warning messages
+--kokkos-print-configuration | print configuration
+--kokkos-tune-internals      | allow Kokkos to autotune policies and declare tuning features through the tuning system. If left off, Kokkos uses heuristics.
+--kokkos-num-threads=INT     | specify total number of threads to use for parallel regions on the host
+--kokkos-map-device-id-by=(random\|mpi_rank) | strategy to select device-id automatically from available devices: random or mpi_rank<sup>2</sup>
+--kokkos-tools-libs=STR      | specify which of the tools to use. Must either be full path to library or name of library if the path is present in the runtime library search path (e.g. LD_LIBRARY_PATH)
+--kokkos-tools-help          | query the (loaded) kokkos-tool for its command-line option support (which should then be passed via --kokkos-tools-args="...")
+--kokkos-tools-args=STR      | a single (quoted) string of options which will be whitespace delimited and passed to the loaded kokkos-tool as command-line arguments. E.g. `<EXE> --kokkos-tools-args="-c input.txt"` will pass `<EXE> -c input.txt` as argc/argv to tool
+
+When passing a boolean as a string, the acceptable values are:
+ - true, yes, 1
+ - false, no, 0
+The values are case-insensitives.
```

Comment:
Spelling error; needs to be "case insensitive."  No hyphen needed in this instance because "case insensitive" is not modifying noun.